frontend/.env

```
@@ -0,0 +1,3 @@
+﻿@
```

dotenv-linter (3.3.0)

[warning] [KeyWithoutValue] The ﻿@ key should be with a value or have an equal sign

(KeyWithoutValue)